proposals/0245-sharing-wifi-ssid-and-password.md

```diff
@@ -1,13 +1,13 @@
 # WiFi Transport: Sharing SSID and password with Mobile Proxy
 
-* Proposal: [SDL-0245](0245-sharing-wifi-ssid-and-password.md)
+* Proposal: [SDL-NNNN](0245-sharing-wifi-ssid-and-password.md)
 * Author: [Ashwin Karemore](https://github.com/ashwink11)

```

Comment:
Please add your name here as well.

Reply:
Fixed in fb4e2a7159168941502b74c6586e06ce38f8fca7

Reply:
Are you taking over this proposal from Ashwin?  If so, I'll need someone from Ford to confirm.  Otherwise, please add yourself as an author, but keep Ashwin too, so that you are both listed.

Reply:
@LuxoftAKutsan yes, if someone from Ford can please provide confirmation that the author should change from Ashwin to you, that would be great.  Thanks!

Reply:
Hi @theresalech, @LuxoftAKutsan did a lot of work on this proposal, while I was away. If it's possible to change the author as per the SDLC process, I confirm that @LuxoftAKutsan is the Author of this proposal.